src/sentry/static/sentry/app/views/settings/organizationAuditLog/auditLogList.jsx

```diff
@@ -110,7 +115,15 @@ class AuditLogList extends React.Component {
                       </NameContainer>
                     </UserInfo>
                     <Box w={150}>{entry.event}</Box>
-                    <Box w={130}>{entry.ipAddress}</Box>
+                    <Box w={130}>
+                      {entry.ipAddress && entry.ipAddress.length > ipv4Length ? (
+                        <Tooltip title={entry.ipAddress}>
```

Comment:
You could simplify this a bit and always render with `<Tooltip>` and `<OverflowBox>`, but disable `<Tooltip>` if ipv4.

Reply:
I like that approach better.

Reply:
I like that approach better.